aws/data_source_aws_organizations_delegated_services_test.go

```diff
@@ -0,0 +1,44 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func TestAccDataSourceAwsOrganizationsDelegatedServices_basic(t *testing.T) {
+	var providers []*schema.Provider
+	dataSourceName := "data.aws_organizations_delegated_services.test"
+	dataSourceIdentity := "data.aws_caller_identity.delegated"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccAlternateAccountPreCheck(t)
+		},
+		ErrorCheck:        testAccErrorCheck(t, organizations.EndpointsID),
+		ProviderFactories: testAccProviderFactoriesAlternate(&providers),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAwsOrganizationsDelegatedServicesConfig(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrPair(dataSourceName, "account_id", dataSourceIdentity, "account_id"),
```

Comment:
This should confirm the number of delegated services

Reply:
The `_basic` case should delegate one service. The zero-case should be a separate test. It would also be good to have a test case that has multiple delegated services.